Moshiur-cy/cleanup-db.js

```diff
@@ -0,0 +1,33 @@
+
+var sql = require("mssql");
+var _ = require('lodash');
+const config = require("./cypress/config/config")
+
+//1
+function executeStoredProc() {

```

Comment:
Trying to review

Reply:
Awesome. You are doing great! Keep up the good work.